editors/utils/reducer.ts

```diff
@@ -49,6 +49,15 @@ export function useDocumentReducer<State, A extends Action, LocalState>(
         setState(_state => {
             try {
                 const newState = reducer(_state, action);
+
+                const operation =
+                    newState.operations[action.scope].slice(-1)[0];
+
+                if (operation.error) {
```

Comment:
Let's reuse the error, might be helpful on Sentry to detect it's the same.
```
const error = new Error(operation.error);
onError?.(error);
onErrorCallback?.(error);
```

Reply:
added fix here [287c8b3](https://github.com/powerhouse-inc/document-model-libs/pull/145/commits/287c8b3d7fb7dc569bf094ef5eea3592fd877047)